UI/html/createblogs.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width", initial-scale="1.0">
+    <title>Create blog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="createblogs.css">
```

Comment:
- This ref is the same as the two previous PRs, this means that the CSS file is in the same folder as this HTML file. Hit me up on slack if you need some help on that.